### DISCUSSION TITLE
Download Failed from www6.f2movies.to

### DISCUSSION THREAD

**Discussion by MarcusAureliusRhodes** [aclap-dev/video-downloadhelper#627] [Help and Q&A]
### Your question

{
  "version": "9.0.1.18",
  "target": "google",
  "channel": "stable",
  "page_url": "https://www6.f2movies.to/watch-movie/watchmen-18976.5349586",
  "error": {
    "details": "SideDownload error: : [https @ 00000226a5c00040] Stream ends prematurely at 1585470, should be 2232312\r\n[aac_adtstoasc @ 00000226a60cf340] Error parsing ADTS frame header!\r\n[mp4 @ 00000226a6145900] Error applying bitstream filters to an output packet for stream #1: Invalid data found when processing input\r\n[aost#0:1/copy @ 00000226a673adc0] Error submitting a packet to the muxer: Invalid data found when processing input\r\n[out#0/mp4 @ 00000226a6a4a5c0] Error muxing a packet\r\n",
    "downloadable_id": "downloadable_8549836821680416",
    "error": "coapp_failure",
    "id": "coapp_failure_d347da2e-b61e-4fb5-87ae-36fdbaa184dd"
  }
}

### Addon & CoApp version and details

version: 9.0.1.18
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"1cd30ba7ff1445b2"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\Marcus\\dwhelper",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": false,
    "open_count_store": 11
}

**Comment by Mattmjs** [comment]
It started downloading and I quit at 4% , so it looks to work on Firefox. When does the error occur ?
